src/test/scala/com/signavio/loadtest/processes/Model.scala

```
@@ -8,28 +8,41 @@ import io.gatling.http.Predef._
 
 object Model {
 
-  val sentHeaders = Map(
+  private val publishedModeHeader = "x-signavio-rev-mode" -> "PUBLISHED"
+
+  private val sentHeaders = Map(
     "Connection" -> "keep-alive",
-    "Origin" -> s"$app_url",
     "Content-Type" -> "application/x-www-form-urlencoded",
-    "User-Agent" -> "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36",
-    "Sec-Fetch-Dest" -> "document",
-    "Accept" -> "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9",
-    "Sec-Fetch-Site" -> "same-origin",
-    "Sec-Fetch-Mode" -> "navigate",
-    "Sec-Fetch-User" -> "?1",
-    "Referer" -> s"$app_url/p/explorer",
-    "Accept-Encoding" -> "gzip, deflate, br",
+    "Accept" -> "application/json",
+    "Accept-Encoding" -> "gzip, deflate",
     "Accept-Language" -> "en-GB,en-US;q=0.9,en;q=0.8",
     s"$csrfProtectionTokenHeaderName" -> s"$csrfProtectionTokenValue",
-    "x-signavio-rev-mode" -> "PUBLISHED"
   )
 
-  def openModel(modelId: String): ChainBuilder = {
-    exec(http("Open Model")
-      .get(s"$app_url/p/model/$modelId")
+  def openModel(modelId: String, repeats: Int = 1): ChainBuilder = {
+    repeat(repeats) {
+      exec(http("Open Model")
+        .get(s"$app_url/p/model/$modelId")
+        .headers(sentHeaders + publishedModeHeader)
+        .check(regex("DMN Model").exists)
+        .check(status is 200)).pause(1)
```

Note to self: with the JSON API, we can be confident that this check will fail if the model is absent etc - unlike with the HTML API (which likely always returns HTTP 200)